gotocompany/optimus/core/v1beta1/job_spec.proto

```diff
@@ -452,8 +452,11 @@ message JobSpecificationResponse {
 
 message ReplaceAllJobSpecificationsRequest {
   string project_name = 1;
-  string namespace_name = 2;
-  repeated JobSpecification jobs = 3;
+  reserved 2 to 3;
+  message jobList {
```

Comment:
I think we don't need any grouping, the request can be same as CreateJobSpecificationRequest